Sources/Sentry/SentrySystemEventsBreadcrumbs.m

```diff
@@ -0,0 +1,174 @@
+#import "SentrySystemEventsBreadcrumbs.h"
+#import "SentrySDK.h"
+#import "SentryLog.h"
+
+// all those notifications are not available for tvOS
+#if TARGET_OS_IOS
+#    import <UIKit/UIKit.h>
+#endif
+
+@implementation SentrySystemEventsBreadcrumbs
+
+- (void)start
+{
+#if TARGET_OS_IOS
+    UIDevice *currentDevice = [UIDevice currentDevice];
+    if (currentDevice == nil) {
+        [SentryLog logWithMessage:@"currentDevice is null, it won't be able to record battery breadcrumbs."
+                         andLevel:kSentryLogLevelDebug];
+        return;
```

Comment:
`h`: Why return when `[self initKeyboardVisibilityObserver];` and `[self initScreenshotObserver];` Would work fine?

Reply:
ops, good catch